extension.js

```diff
@@ -14,6 +12,10 @@ const Shell = imports.gi.Shell;
 const Signals = imports.signals;
 const St = imports.gi.St;
 
+const Gst = try_to_import_or_return_null(() => { return imports.gi.Gst; });
```

Comment:
I have to think it over.

Reply:
I might miss some clever trick, but to prevent:
> ReferenceError: Gst is not defined

it would have to use multi-assignment on values returned from the method and I'm not sure if it would increase readability.

I also cannot move it after Extension as it would not be initialized.

Reply:
I did this:
```
const Extension = ExtensionUtils.getCurrentExtension();

let isPlayingSoundSupported = true;
let Gst;
let GstAudio
try {
    Gst = imports.gi.Gst;
    GstAudio = imports.gi.GstAudio;
} catch(e) {
    isPlayingSoundSupported = false;
    log(`${Extension.metadata.uuid}: Unable to import sound module. Playing sound is not available. Is GStremer package installed?`);
    log(`${Extension.metadata.uuid}: ${e}`);
}
```
Which I thought was a bit cleaner.

Reply:
I don''t know why I had `ReferenceError: Gst is not defined` before :-/. When I switched the order of statements again it works. I changed them.

However, personally, I prefer using constants instead of variables. The method name is self explaining. Nevertheless, if @wbolster prefers the construction you proposed, we can change it.

Reply:
no strong preference: i like `const` but i also like `non-nullable` types 😉 so it's a trade-off. anyway, you can leave it as-is 👍🏼

Reply:
This was more a personal preference so you can keep it as it is. I don't see any actual issues with it (it's just not how I would have done in🙂).